lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -127,6 +127,119 @@ def onReplace(self, *args):
         # Nothing needed here since we don't create any UI.
         pass
 
+class MaterialCustomControl(object):
```

Comment:
Is there a reason this class doesn't derive from `AttributeCustomControl`?

Also is there a reason for not placing this custom control in its own file (which we've done for others)?

Reply:
I can fix both of these in the next PR. It is based on this one and awaiting his one to be open

Reply:
(Not deriving from AttributeCustomControl did not seem to affect anything)

Reply:
(The others also don't seem to derive from AttributeCustomControl?)